config/checkstyle_checks.xml

```diff
@@ -281,6 +281,10 @@
     <module name="ReturnCount">
       <property name="maxForVoid" value="0"/>
     </module>
+    <module name="ReturnCount">
```

Comment:
No. While it is a separate module we can suppress violations in some files for `max=1`, but still enforce `max=2` in the same files. If the modules are combined we will suppress violations of `ReturnCount` completely.